notes-service/src/main/java/org/exoplatform/wiki/jpa/JPADataStorage.java

```diff
@@ -1595,7 +1595,24 @@ public PageVersion getPageVersionById(long versionId) {
    * {@inheritDoc}
    */
   @Override
-  public List<DraftPage> getDraftsOfWiki(String wikiOwner, String wikiType) {
-    return convertDraftPageEntitiesToDraftPages(draftPageDAO.findDraftsOfWiki(wikiOwner, wikiType));
+  public List<DraftPage> getDraftsOfWiki(String wikiOwner, String wikiType, String wikiHome) {
+    // The Note API allows multiple home pages to be created within the wiki.
```

Comment:
normally only one home note is allowed for a wiki

Reply:
what is the Articles page created in content addon ?